### PR TITLE
Cb form components

### LINKED ISSUE #299: Style cb login and new password views

### PR DESCRIPTION
**Pull request by eliascodes**
Fixes #299 

OK sticking this up now because better for us to have consistent base components to work from.

Note that the error handling on the form components is NOT COMPLETE...that is, it's basically broken. This is because I want to update those endpoints to move towards the new API. I will do this now while the PR is being reviewed and open a new one once I'm done.

HMU ✌️

### CODE REVIEW

File: react-backend/client/src/cb-admin/pages/ConfirmPassword.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import LabelledInput from '../../shared/components/form/LabelledInput';
+import { FlexContainerCol } from '../../shared/components/layout/base';
+
+
+const SubmitButton = PrimaryButton.extend`
+  height: 4em;
+  width: 90%;
+`;
+
+const CenteredParagraph = Paragraph.extend`
```
Comment (RogeredBacon) [inline react-backend/client/src/cb-admin/pages/ConfirmPassword.js]: You use the `.extend` here but not in ResetPassword or ForgotPassword, is that intentional?

Comment (eliascodes) [reply react-backend/client/src/cb-admin/pages/ConfirmPassword.js]: No, just oversight, will fix it in the subsequent PR

**Comment by astroash** [comment]
@eliascodes you've removed a lot of auth components in this PR. Is it going to break any pages that others are working on?

**Comment by eliascodes** [comment]
@astroash shouldn't do as long as you enter the right password :wink:

**Pull request merged**